src/modules/atommodel/serializer/RadixSerializer.ts

```diff
@@ -91,7 +65,7 @@ export class RadixSerializer {
      */
     public static registerClass(serializer: string) {
         return (constructor: typeof RadixSerializableObject) => {
-            const hashedSerializer = javaHashCode(serializer)
+            const hashedSerializer = serializer
```

Comment:
Nitpicking on the naming of the constant (really nothing to address unless you're making other changes in this file), but is `hashedSerializer` really hashed now?